resources/master-kubelet-conf.yaml

```diff
@@ -15,6 +15,12 @@ cgroupDriver: systemd
 clusterDNS:${cluster_dns}
 clusterDomain: "cluster.local"
 ${feature_gates == "" ? "" : "featureGates:\n  ${feature_gates}"}
+containerRuntimeEndpoint: "unix:///run/containerd/containerd.sock"
+registerWithTaints:
+  - key: "node-role.kubernetes.io/master"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/control-plane"
```

Comment:
Everything scheduled on master nodes will need to be patched in advance to comply with that, like: https://github.com/utilitywarehouse/system-manifests/pull/188